solutions/silver/cf-839C.mdx

```diff
@@ -97,6 +97,65 @@ int main() {
 
 </CPPSection>
 
+<JavaSection>
+```java
+import java.util.*;
+
+public class Journey {
+	public static double ans = 0;
```

Comment:
wait...double??

Reply:
The answer will be a decimal value, that is the reason double is used.